src/components/TextInput/BaseTextInput.js

```diff
@@ -40,6 +42,8 @@ function BaseTextInput(props) {
     const input = useRef(null);
     const isLabelActive = useRef(initialActiveLabel);
 
+    const operatingSystem = getOperatingSystem();
```

Comment:
we should not use such logic - i will suggest to move line 375 in a function - which will be defined in 2 files like index and index.native - for native we should not make any changes to current file - because everything is working fine